website/docs/understanding_ferretdb.md

```diff
@@ -56,6 +56,69 @@ For complex documents, you can nest objects (subdocuments) inside a document.
 
 In the example above, the `name` field is a subdocument embedded into a document.
 
+## Dot Notation
+
+Dot notations `(.)` are used to specify the elements in an array, the objects in a document, or the values in an embedded document.
```

Comment:
What is the difference between objects in document and values in the embedded document?